backend/molgenis-emx2-graphgenome/src/main/java/org/molgenis/emx2/graphgenome/GraphGenome.java

```diff
@@ -17,8 +17,8 @@
 
 public class GraphGenome extends RDFService {
 
-  public GraphGenome(String requestURL, String format) {
-    super(requestURL, format);
+  public GraphGenome(String baseURL, String rdfAPIpath, String format) {
+    super(baseURL, rdfAPIpath, format);
```

Comment:
why both paths? Probably will understand it later :-)

Reply:
This has to do with the way that MOLGENIS is setting up the paths. Everything is <baseUrl> / <schema> / <app or api endpoint>.